arch/arm/boot/dts/overlays/ssd1306-spi-overlay.dts

```diff
@@ -73,9 +73,10 @@
 	__overrides__ {
 		speed		= <&ssd1306>,"spi-max-frequency:0";
 		rotate		= <&ssd1306>,"rotate:0";
-		fps		= <&ssd1306>,"fps:0";
+		fps			= <&ssd1306>,"fps:0";
```

Comment:
It looks like the stray whitespace found another home.

Reply:
gnaaa I hate tabs ;)